css/utilities.css

```diff
@@ -8,6 +8,17 @@
     justify-content: space-between;
 }
 
+.title {
+    font-size: var(--font-size-large);
+    color: var(--color-title);
+}
+
+.empty-container-text {
+    display: block !important;
```

Comment:
You shouldn't use `!important` -- `.empty-container` sounds more like a component rather than a utility -- same goes for `title` and `.button`.